### DISCUSSION TITLE
如何通过禁用部分功能达到速度的提升，我现在启动要好几秒

### DISCUSSION THREAD

**Discussion by qinjianwei** [ayamir/nvimdots#781] [Q&A] [answered]
![image](https://github.com/ayamir/nvimdots/assets/17981557/fed82031-2e4d-40f1-b0c5-e400e8f62c96)

**Comment by ayamir** [comment, marked as answer]
https://github.com/ayamir/nvimdots/blob/ec9143749fff8e1cd47e6559dc3315d7690a90a8/lua/core/settings.lua#L34-L37

**Comment by Jint-lzxy** [comment]
Closed due to inactivity. Please reactivate if further support is needed.
